documentation/en/chapter_3.md

```diff
@@ -0,0 +1,5 @@
+# 3. Modificaciones propuestas
+
+The goal is to implement the current version of the AODVv2 with no major changes in an embedded system with limited resources to analyze the behavior of the network while working the routing protocol. It's also part of the goal to build the hardware to obtain and analyze the results in real environments.  
+
+If the protocol's working parameters change, it may improve or downgrade the performance of the nodes in the network. For instance, the TTL (Time To Live), which indicates the lifespan of a route stored inside a router, is a very important parameter to take into account that may have a strong impact on the performance of the network, and could change depending on the size of the network. [2]
```

Comment:
```suggestion
If the protocol's working parameters change, it may improve or downgrade the performance of the nodes in the network. For instance, the TTL (Time To Live), which indicates the lifespan of a route stored inside a router, is a very important parameter to take into account that it would may have a strong impact on the performance of the network, and could change depending on the size of the network. [2]
```

Reply:
>  is a very important parameter to take into account that it would have a strong impact over network performance, and it could change based on network size.

deleted the -may-

Reply:
So, I was checking for the use of the word 'may' and 'might'. For instance, 'may' indicates something likely to happen, while 'might' indicates something less likely to happen. We can stay with the 'may' as stated in the Spanish version or submit a change.